benches/speed_tests.rs

```diff
@@ -230,7 +229,10 @@ pub fn prio3_client(c: &mut Criterion) {
     });
 
     let len = 1000;
-    let prio3 = Prio3Aes128CountVec::new(num_shares, len).unwrap();
+    let prio3 = new_prio3_aes128_count_vec::<ParallelSum<F, BlindPolyEval<F>>, PrgAes128, 16>(
+        num_shares, len,
+    )
+    .unwrap();
     let measurement = vec![0; len];
     println!(
         "prio3 countvec ({} len) size = {}",
```

Comment:
Should `Prio3Aes128CountVecMultithreaded` (see below) have its own constructor, to match the others?